### PR TITLE
Report code coverage results to codecov

### PR DESCRIPTION
**Pull request by armenzg**

### CODE REVIEW

**Comment by armenzg** [comment]
It's showing up in https://codecov.io/gh/mozilla/treeherder/commits
Their system will show more info once their build system has processed it.

File: .gitignore [hunk 1]
```diff
@@ -32,3 +32,6 @@ celerybeat-schedule
 
 #docker custom environment
 .env
+
+# Ignore coverage results
+coverage
```
Comment (camd) [inline .gitignore]: Nit: need return at end of file...

**Comment by armenzg** [comment]
> Thanks for setting this up. Will there be an automatic code coverage comment on every pr that's opened?

Yes, we will need to configure it to determine what % decreases should cause a "red" status vs just "warning".

Any thoughts? We can start with any decrease more than 2% should cause a "red"?

**Comment by camd** [comment]
@armenzg : Sure, let's go with 2% for now.   This is really great!  I'm doing some learning on how to use it for now...  :)

**Pull request merged**